soroban-built-in-token-spec/src/lib.rs

```diff
@@ -0,0 +1,152 @@
+#![no_std]
```

Comment:
Could you add a comment to the top of this file linking to where these types were copied from, as well as the Git revision they were copied from, as well as what is modified? So that if someone else needs to do same, they know what steps to follow.

Reply:
Added a comment.